PeriodogramAnalysis/test_power_density_statistics.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+from miv.core.datatype import Signal
+from power_density_statistics import (
+    SpectrumAnalysisPeriodogram,
+    SpectrumAnalysisMultitaper,
+    SpectrumAnalysisWelch,
+)
+
+
+def mock_signal_generator():
+    data1 = np.array(
+        [
+            np.sin(2 * np.pi * 5 * np.linspace(0, 1, 100)),  # 5 Hz
+            np.sin(2 * np.pi * 10 * np.linspace(0, 1, 100)),  # 10 Hz
+        ]
+    ).T
+    timestamps = np.linspace(0, 1, 100)
+    signal1 = Signal(data=data1, timestamps=timestamps, rate=100.0)
+    yield signal1
+
+    data2 = np.array(
+        [
+            np.sin(2 * np.pi * 15 * np.linspace(0, 1, 200)),  # 15 Hz
+            np.sin(2 * np.pi * 20 * np.linspace(0, 1, 200)),  # 20 Hz
+        ]
+    ).T
+    timestamps = np.linspace(0, 1, 100)
+    signal2 = Signal(data=data2, timestamps=timestamps, rate=100.0)
+    yield signal2
+
+
+def test_SpectrumAnalysisWelch(tmp_path):
```

Comment:
I don't get these tests. You created mock signals, but all assertion you are making in the test is whether `psd_dict` is dictionary and if `freqs` and `psd` is in the result??

Reply:
Yeah, it is weird... have added numerical tests in these commits: [d7a26a7] [424b207]